src/pages/ReimbursementAccount/BusinessInfo/substeps/TaxIdBusiness.tsx

```diff
@@ -35,7 +26,15 @@ const validate = (values) => {
     return errors;
 };
 
-function TaxIdBusiness({reimbursementAccount, onNext, isEditing}) {
+type TaxIdBusinessOnyxProps = {
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+};
+
+type TaxIdBusinessProps = {
+    reimbursementAccount: ReimbursementAccount;
+} & SubStepProps;
```

Comment:
Move types to the top of the file
```suggestion
type TaxIdBusinessProps = TaxIdBusinessOnyxProps & SubStepProps;
```